test/lib/mayaUsd/fileio/testImportWithNamespace.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env mayapy
+#
+# Copyright 2022 Autodesk
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from maya import cmds
+from maya import standalone
+
+from pxr import Usd, UsdGeom
+
+import fixturesUtils
+
+import unittest
+
+class testReadWriteUtils(unittest.TestCase):
```

Comment:
Name of the class seems a bit disconnected with the actual test about namespace.

Reply:
Ack!  Sorry about that...  Copy-paste damage.  Will fix once you're done with the review and the build has completed.